src/plugins/ssu_path/mod.rs

```diff
@@ -0,0 +1,245 @@
+    }
+}
+
+const ENTRIES_PER_PAGE: usize = 10;
+
+impl SsufidPlugin for SsuPathPlugin {
+    const IDENTIFIER: &'static str = "path.ssu.ac.kr";
+    const TITLE: &'static str = "숭실대학교 SSU-PATH";
+    const DESCRIPTION: &'static str =
+        "숭실대학교 비교과 시스템 SSU-PATH의 비교과 프로그램 정보를 제공합니다.";
+    const BASE_URL: &'static str =
+        "https://path.ssu.ac.kr/ptfol/imng/icmpNsbjtPgm/findIcmpNsbjtPgmList.do";
+
+    async fn crawl(&self, posts_limit: u32) -> Result<Vec<SsufidPost>, PluginError> {
+        info!(
+            "Crawling {} with {} posts limit",
+            SsuPathPlugin::IDENTIFIER,
+            posts_limit
+        );
+        let pages = (posts_limit as usize).div_ceil(ENTRIES_PER_PAGE);
+        info!("Crawling {} pages", pages);
+        let client = self.client().await?;
+        let entries = (1..=pages)
+            .map(|page| entries(&client, page))
+            .collect::<FuturesUnordered<_>>()
+            .try_collect::<Vec<_>>()
+            .await?
+            .into_iter()
+            .flatten()
+            .take(posts_limit as usize)
+            .collect::<Vec<_>>();
+        Ok(entries
+            .iter()
+            .map(|entry| post(&client, entry))
+            .collect::<FuturesUnordered<_>>()
+            .try_collect::<Vec<_>>()
+            .await?)
+    }
+}
+
+const PATH_LIST_URL: &str = "https://path.ssu.ac.kr/ptfol/imng/icmpNsbjtPgm/findIcmpNsbjtPgmList.do?paginationInfo.currentPageNo=";
+
+static ENTRIES_SELECTOR: LazyLock<Selector> =
+    LazyLock::new(|| Selector::parse("div.lica_wrap > ul > li").unwrap());
+
+async fn entries(
+    client: &reqwest::Client,
+    page: usize,
+) -> Result<Vec<SsuPathProgram>, SsuPathPluginError> {
+    let url = format!("{PATH_LIST_URL}{page}");
+    info!("Crawling entries from {}", url);
+    let response = client.get(url).send().await?.text().await?;
+    let document = Html::parse_document(&response);
+    document
+        .select(&ENTRIES_SELECTOR)
+        .map(SsuPathProgram::from_element)
+        .collect::<Result<Vec<SsuPathProgram>, SsuPathPluginError>>()
+}
+
+const PATH_ENTRY_URL: &str =
+    "https://path.ssu.ac.kr/ptfol/imng/icmpNsbjtPgm/findIcmpNsbjtPgmInfo.do?encSddpbSeq=";
+
+async fn post(
+    client: &reqwest::Client,
+    program: &SsuPathProgram,
+) -> Result<SsufidPost, SsuPathPluginError> {
+    info!("Crawling program {}", program.id);
+    let url = format!("{PATH_ENTRY_URL}{}", program.id);
+    let response = client.get(&url).send().await?.text().await?;
+    let document = Html::parse_document(&response);
+    let program_table = SsuPathProgramTable::from_document(&document)?;
+    let course_table = match program.kind {
+        SsuPathProgramKind::Single { .. } => Some(SsuPathCourseTable::from_document(&document)?),
+        _ => None,
+    };
+    let division_table = match program.kind {
+        SsuPathProgramKind::Division { .. } => {
+            Some(SsuPathDivisionTable::from_document(&document)?)
+        }
```

Comment:
`Division`이 튜플 구조체임을 명시적으로 표현하기 위해 `(_)` 형태의 패턴을 사용하는건 어떨까요? 
```suggestion
        SsuPathProgramKind::Division(_) => Some(SsuPathDivisionTable::from_document(&document)?),
```

Reply:
https://github.com/yourssu/ssufid/pull/82/commits/b102616d556465f7053e7bbf189484f818d6e044

반영완료